### DISCUSSION TITLE
Connecting the positive and negative nodes of SD Prompt Saver

### DISCUSSION THREAD

**Discussion by Cahya** [receyuki/comfyui-prompt-reader-node#28] [Q&A]
Where should I connect the positive and negative nodes of SD Prompt Saver? I can't find nodes that can be connected to them. In my workflow, I only use SD Prompt Generator and SD Prompt Saver. I don't use SD Prompt Reader.

**Comment by receyuki** [comment]
Apologies for the late response, I don't check the discussions frequently.

You can link any string format node to the `positive` and `negative.` For instance, usually, you would use CLIP Text Encode to connect the prompt, in that case, you just need to connect the node linked to the `text` on CLIP Text Encode to the `positive` and `negative.`
